geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/FilterHelperTest.scala

```diff
@@ -353,4 +356,15 @@ class FilterHelperTest extends Specification with Mockito with Logging {
       result.asInstanceOf[KnownCost].cost mustEqual 5
     }
   }
+
+  "tryMergeGeoms" should {
+    "intersect BBOX geoms" >> {
```

Comment:
It might be nice to have an additional test that (intentionally) is unable to find an intersection.